okteto-example.yml

```
@@ -1,16 +1,9 @@
 name: xcube-hub-okteto
```

Hi Tonio,

```suggestion:
name: xcube-hub
```

I was wrong yesterday. This line should contain the name of the deployment i.e. `xcube-hub`. Okteto replaces the xcube-hub pod in teh xcube-hub deployment with the okteto pod. This allows you to debug the hub within the K8s infrastructure. Nice but playing around has broken the deployment yesterday.

---

This also has of course implications for dev. Once shoud work in the own minikube or so. Not on the dev cluster except one wants to check something specific.